apps/test-blinky/src/TutorialServo.cpp

```diff
@@ -0,0 +1,42 @@
+//
+//  TutorialServo.cpp
+//  
+//
+//  Created by Ari Wasch on 9/18/20.
+//
+
+#include "TutorialServo.h"
+#include "mbed.h"
+
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0, float minPulsewidthInMs = 1, float maxPulsewidthInMs = 2){
+    m_servoPwmOut(PA_1)
+    m_servoPwmOut.period(0.020);
```

Comment:
this is fine but id just specify the period in the constructor of the pwm object, but up to you